beets/ui/__init__.py

```diff
@@ -1159,10 +1159,13 @@ def _open_library(config):
         )
         lib.get_item(0)  # Test database connection.
     except (sqlite3.OperationalError, sqlite3.DatabaseError):
+        message = ""
         log.debug(u'{}', traceback.format_exc())
-        raise UserError(u"database file {0} could not be opened".format(
+        if e.args[0] == "unable to open database file":
+            message = "It might be a permissions problem."
+        raise UserError(u"database file {0} could not be opened.%s".format(
             util.displayable_path(dbpath)
-        ))
+        )%message)
```

Comment:
Hello,
I' ve tried both ways but i have the same problem.  In the first way i just add in mutate ` raise UserError("unable to open database file. It might be a permissions problem")` and in the second way i wrote in `main` this:
```
    except sqlite3.OperationalError as e:
        if e.args[0] == "unable to open database file":
           print("unable to open database file. It might be a permissions problem")
```
I wrote also a new error exception class
```
class AccessFileError(Exception):
     """Permission Exception. Exception that Specifically inidcates the
     possibility of a permissions problem on database file.
     """
```
In order to work with this way i have to raise an `AccessFileError` in mutate and catch it in the `main`?

Thank you very much!